docs/tutorial-module/managing-access/index.rst

```diff
@@ -0,0 +1,166 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2018 CERN.
+    Copyright (C) 2018 Northwestern University, Feinberg School of Medicine, Galter Health Sciences Library.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+.. _access_index:
+
+Overview
+========
+
+The cookiecutter instance that you have created when following :ref:`quickstart` comes with ``Custom`` data model
+included in the instance. The data model provides tools for CRUD operations and search. It leaves the flexibility
```

Comment:
I think end of sentence `included in the instance` is not needed. What do you think?